src/ucp/core/ucp_request.h

```diff
@@ -169,6 +170,42 @@ typedef struct ucp_recv_desc {
 extern ucs_mpool_ops_t ucp_request_mpool_ops;
 
 
+static UCS_F_ALWAYS_INLINE ucp_recv_desc_t *
+ucp_recv_desc_get(ucp_worker_t *worker, void *data, void *desc, size_t length,
+                  uint16_t hdr_len, uint16_t flags)
+{
+    ucp_recv_desc_t *rdesc = (ucp_recv_desc_t *)data - 1;
+    if (rdesc == desc) {
+        rdesc->flags = flags | UCP_RECV_DESC_FLAG_UCT_DESC;
+    } else {
+        rdesc = (ucp_recv_desc_t *)ucs_mpool_get_inline(&worker->am_mp);
```

Comment:
is it coming from gtest compilation?
is ucp_request.h included from C++ file? if so can we avoid it?